crates/core/src/client/config.rs

```diff
@@ -36,6 +37,7 @@ impl Network {
         match self {
             Network::Katana => Ok(Url::parse(KATANA_RPC_URL)?),
             Network::Madara => Ok(Url::parse(MADARA_RPC_URL)?),
+            Network::Sharingan => Ok(Url::parse(std::env::var("SHARINGAN_RPC_URL").unwrap().as_str())?),
```

Comment:
Isnt it possible to have the same format as how we access Madara and Katana Url?

Reply:
I opted to have this as an environment variable and not a constant like the other two as it can change depending on the deployment i.e. part of the madara stack docker compose (http://host.docker.internal:9944), stand-alone (https://sharingan.madara.zone or something else in the future)